src/components/PostInfo/PostInfo.jsx

```diff
@@ -1,3 +1,38 @@
-export const PostInfo = () => (
-  <>Put the post here</>
-);
+import './PostInfo.scss';
+import { CommentList } from '../CommentList';
+import { UserInfo } from '../UserInfo';
+
+export const PostInfo = ({ post }) => {
+  const {
+    user,
+    title,
+    body,
+    comments,
+  } = post;
+
+  return (
+    <div className="PostInfo">
+      <div className="PostInfo__header">
+        <h3 className="PostInfo__title">{title}</h3>
+
+        <p>
+          {' Posted by  '}
+
+          <UserInfo user={user} />
+        </p>
+      </div>
+
+      <p className="PostInfo__body">
+        {body}
+      </p>
+
+      <hr />
+
+      {
+        comments.length !== 0
```

Comment:
```suggestion
        comments.length
```
I think it will also work properly without it